.test-infra/jenkins/job_BenchmarkTests_CloudML_Python.groovy

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as commonJobProperties
+import PhraseTriggeringPostCommitBuilder
+import CronJobBuilder
+
+def cloudMLJob = { scope ->
+  scope.description('Runs the TFT Criteo Examples on the Dataflow runner.')
+
+  // Set common parameters.
+  commonJobProperties.setTopLevelMainJobProperties(scope)
+
+  // Gradle goals for this job.
+  scope.steps {
+    gradle {
+      rootBuildScriptDir(commonJobProperties.checkoutDir)
+      commonJobProperties.setGradleSwitches(delegate, 'master', 360)
+      tasks(':sdks:python:test-suites:dataflow:tftTests')
+    }
+  }
+}
```

Comment:
Should we also be running this as a normal postcommit like https://github.com/apache/beam/blob/853fdd38aafda9d01648bd3733103b1a34a044a1/.test-infra/jenkins/job_InferenceBenchmarkTests_Python.groovy#L192

Reply:
I am not sure. These run for 4 hours. I thought daily once as a cron job seems good enough but I can add a trigger command.

Reply:
> These run for 4 hours

I think that's probably ok; if it overwhelms our CI we can revisit, but that's not out of band with some of our other postcommits. The advantage of having it as a postcommit is that you can more easily correlate regressions to changes in the codebase